smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/OpenApiProtocol.java

```diff
@@ -62,6 +62,15 @@ default void updateDefaultSettings(Model model, OpenApiConfig config) {
      * Creates an operation entry, including the method, URI, and operation
      * object builder.
      *
+     * The returned operation object builder should contain protocol-specific fields of an
+     * OpenAPI Operation object, such as:
+     * - parameters, requestBody, responses
```

Comment:
note that Java doesn't support markdown in comments. You should either use HTML here or just omit the attempted styling.

Reply:
omitted styling.